libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -189,6 +200,19 @@ void AP_OSD_Screen::draw_gspeed(uint8_t x, uint8_t y)
     backend->write(x, y, false, "%3.0f%c", v, SYM_KMH);
 }
 
+void AP_OSD_Screen::draw_gcourse(uint8_t x, uint8_t y)
+{
+    AP_GPS & gps = AP::gps();
```

Comment:
we should use the AHRS ground course for this, it will be a lot less noisy

Reply:
AHRS should be used for gspeed too.